.github/workflows/generator_generic_slsa3.yml

```diff
@@ -154,9 +158,36 @@ jobs:
           path: "${{ inputs.attestation-name }}"
           sha256: "${{ needs.generator.outputs.attestation-sha256 }}"
 
-      - name: Release
+      - name: Upload provenance
```

Comment:
I'm a little worried people will assume they need `upload-assets` to be true when they specify an `upload-url` 

Maybe we can clarify that in the description? Or allow that case, and only run this job when `upload-url` is nil.

Reply:
I agree it's confusing. How about requiring `upload-assets` be true *always* to an upload to occur? 

If `upload-assets`, the upload will occur automatically on new tags. If not new tag, users need this optional `upload-url` to indicate where to upload. 

Wdut?

Reply:
I've updated the doc to show what it would look like for the suggestion I made.